src/utils/PrivateRoute.js

```diff
@@ -8,7 +8,7 @@ const PrivateRoute = ({ component: Component, ...rest }) => {
     <Route
       {...rest}
       render={(props) =>
-        session ? <Component {...props} /> : <Redirect to="/" />
+        session ? <Component {...props} /> : <Navigate to="/" />
```

Comment:
PrivateRoute would need some changes

I was looking at the docs example so could  be something like this 

```
const PrivateRoute({ children }) {
  const { session } = useAuth();
  if (!session) {
    return <Navigate to="/login" />;
  }
  return children;
}

export default PrivateRoute;
```
[example link](https://stackblitz.com/github/remix-run/react-router/tree/main/examples/auth?file=src/App.tsx)